ui/library/src/components/DocumentWrapper.tsx

```diff
@@ -51,11 +55,34 @@ export const DocumentWrapper: React.FunctionComponent<Props> = ({
     setIsLoading(false);
   }, []);
 
+  const onItemClicked = (param: any): void => {
+    if (!pdfDocProxy) {
+      return;
+    }
+
+    // Scroll to the destination of the item
+    pdfDocProxy.getDestination(param.dest).then(destArray => {
+      if (!destArray) {
+        return;
+      }
+
+      const [ref, , , bottomPoints] = destArray;
+      pdfDocProxy.getPageIndex(new Ref(ref)).then(refInfo => {
+        scrollToPosition(parseInt(refInfo.toString()), 0, bottomPoints, rotation);
+      });
+    });
+  };
+
   return (
     <Document
       options={{ cMapUrl: 'cmaps/', cMapPacked: true }}
       onLoadError={onPdfLoadError}
       onLoadSuccess={onPdfLoadSuccess}
+      externalLinkTarget="_blank"
```

Comment:
`rel` is set to `'noreferrer'` by default which is the same as the default setting in `ExternalLink.jsx` so I didn't change it.
<img width="1358" alt="Screen Shot 2022-04-21 at 16 37 55" src="https://user-images.githubusercontent.com/84034381/164568019-77a8ce04-27e4-4a13-bf17-dd632cc906ea.png">